src/modules/0.0.13/aws_dynamo/migration/1658157120785-aws_dynamo.ts

```diff
@@ -1,7 +1,7 @@
 import {MigrationInterface, QueryRunner} from "typeorm";
 
-export class awsDynamo1657988050211 implements MigrationInterface {
-    name = 'awsDynamo1657988050211'
+export class awsDynamo1658157120785 implements MigrationInterface {
+    name = 'awsDynamo1658157120785'
 
```

Comment:
Oh interesting the migration did not change, I wasn't sure what was the default.

Reply:
Apparently for JSON it's `not null` but for others it is `null`? It would be nice if TypeORM was more consistent here.